test/specs/helpers.spec.js

```diff
@@ -126,7 +126,7 @@ define([
                 d3.select('.test-container svg').attr('width', 200);
             });
 
-            describe('convertSvgToHtml', () => {
```

Comment:
It has something to do with the rendering of the HTML in phantomJS.
We could either explore this issue or try the xvfb virtual desktop option on travis (removing phantomJS)